docker/lb/README.md

```diff
@@ -0,0 +1,27 @@
+# nginx based Load Balancer for Avalon
+
+This shows how to create nginx based Load Balancer to forward the requests
+coming from clients to one of the many avalon-listeners running in backend.
+
+## Running
+
+1. Build and run Avalon components.
+    ```bash
+    docker-compose -f docker/lb/docker-compose-lb.yaml up --build
+
+    The above command is going to start nginx based service (avalon-lb) along
+    with other avalon services. The client is going to submit the transaction
+    to avalon-lb Load Balancer and this load balancer is going to forward the
+    requests to 3 instances of avalon-listeners in the backend in round-robin
+    fashion.
+    ```
+
+2. Start transaction from client.
+    ```
+    # Go to shell container
+    docker exec -it avalon-shell bash
+ 
+    # run the transaction
+    cd examples/apps/echo/client
+    ./echo_client.py -m "Hello world" -s "http://avalon-lb" -dh
```

Comment:
Can we use generic client instead of echo client. Echo client will be deprecated soon and replaced with generic client